src/Tests/Moryx.ControlSystem.Tests/EnumInstructionResultTests.cs

```diff
@@ -25,6 +25,7 @@ public void GetAllValuesIfNoResultIsDecorated()
 
             // Assert
             Assert.AreEqual(2, instructionResult.Results.Count(), "There should be 2 results because all of the results are not decorated");
+            Assert.AreEqual(2, instructionResult.PossibleResults.Count(), "There should be 2 results because all of the results are not decorated");
```

Comment:
Did you intend to duplicate the line here?

Reply:
Kinda, it checks for the new property on the instruction

Reply:
Ahh I missed that detail 👌